src/core/env/src/main/scala/FileUtilities.scala

```diff
@@ -30,10 +30,12 @@ object FileUtilities {
     }
   }
 
-  def delTree(file: File): Boolean =
-    if (!file.exists) true
-    else { if (file.isDirectory) file.listFiles.forall(delTree)
-           file.delete }
+  private[microsoft] def delTree(file: File): Unit =
+    if (!file.exists) {
```

Comment:
I was just reminded of this too but it's more idiomatic to use require for IllegalArgumentExceptions